cobra/io/sbml.py

```diff
@@ -1288,7 +1289,11 @@ def _sbase_notes_dict(sbase, notes):
 
 In the current stage the new annotation format is not completely supported yet.
 """
-URL_IDENTIFIERS_PATTERN = re.compile(r"^https?://identifiers.org/(.+?)/(.+)")  # noqa: E501
+
+URL_IDENTIFIERS_PATTERN1 = re.compile(r"^https?://identifiers.org/(.+?)/(.+)")  # noqa: E501
```

Comment:
Might be wrapped into one pattern if that helps?

```suggestion
URL_IDENTIFIERS_PATTERN1 = re.compile(r"^https?://identifiers.org/(.+?)[:/](.+)")  # noqa: E501
```

Reply:
You can also add a `re.I` flag for ignoring case since URLs are not case sensitive.

Reply:
Will make the change. Regular expressions are my weakness, always a trial and error for me.